usermods/Ble2Json_v2/ble.h

```diff
@@ -0,0 +1,100 @@
+#pragma once
+
+#include "wled.h" 
+#include <BLEDevice.h>
+#include <BLEServer.h>
+#include <BLEUtils.h>
+
+BLEServer* pServer = NULL;
+BLECharacteristic* pCharacteristic = NULL;
+#define SERVICE_UUID        "4fafc201-1fb5-459e-8fcc-c5c9c331914b"
+#define CHARACTERISTIC_UUID "beb5483e-36e1-4688-b7f5-ea07361b26a8"
+#define PASSKEY 999999
+
+/////////////////////
+//BLE Secure Server//
+/////////////////////
+
+class ServerCallback: public BLEServerCallbacks {
+    void onConnect(BLEServer* pServer) {
+      Serial.println(" - ServerCallback - onConnect");
+    };
+
+    void onDisconnect(BLEServer* pServer) {
+      Serial.println(" - ServerCallback - onDisconnect");
+    }
+};
+
+class SecurityCallback : public BLESecurityCallbacks {
+
+  uint32_t onPassKeyRequest(){
+    return 000000;
```

Comment:
previously you use `#define PASSKEY 999999`. Is it correct to return 0 here? Is there any special reason to write `000000` instead of 0? or did you intend to return a string, or maybe a hex number?

Reply:
there are two passkeys... one from the server and one from the client... we don't care about the server's passkey, only the one from the client... so returning 0 is fine for the server (esp32 device).... I formatted it as all zeros because I copied this code from somewhere... I can change it to just one zero.  the "real" default passkey is all 9's... but that is another thing one can change in the config (though, prolly should be added to the "secure" config... in the future)